### PR TITLE
Bug 355 google streetviewshows only dead sea location

### PR DESCRIPTION
**Pull request by danielsh28**
added typescipt, but have two issues on that `//ts-ignore` on third party library `react-async-loader` with no typescript declaration files. and `any` type on the google maps global object

### CODE REVIEW

File: src/services/streetView.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// @ts-ignore
+import asyncLoading from 'react-async-loader';
```
Comment (yuvalbl) [inline src/services/streetView.tsx:4]: Even if there is no types file for this package, Why `@ts-ignore` is required?
Is it referring to this issue?
https://github.com/microsoft/typescript/issues/3019

File: src/services/streetView.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+interface ReactStreetviewProps {
```
Comment (yuvalbl) [inline src/services/streetView.tsx:6]: Why not keeping our `IProps` convention?

File: src/services/streetView.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  apiKey: string;
+  streetViewPanoramaOptions: google.maps.StreetViewPanoramaOptions;
+  onPositionChanged: Function;
+  onPovChanged: Function;
+  position: google.maps.LatLng;
+  googleMaps: any;
+}
+
+class ReactStreetview extends React.Component<ReactStreetviewProps> {
```
Comment (yuvalbl) [inline src/services/streetView.tsx:15]: Why not functional component?

Comment (yuvalbl) [reply src/services/streetView.tsx:15]: please add reference to documentation

**Review by yuvalbl — APPROVED** [review]

File: src/services/streetView.tsx (hunk 3 above)
Comment (danielsh28) [reply src/services/streetView.tsx:15]: README  file updated in https://github.com/hasadna/anyway-newsflash-infographics/pull/407

**Pull request merged**